src/blocks/tea_timer.rs

```diff
@@ -77,32 +83,41 @@ pub async fn run(config: &Config, api: &CommonApi) -> Result<()> {
         }
         timer_was_active = is_timer_active;
 
-        let (hours, minutes, seconds) = if is_timer_active {
-            (
-                remaining_time.num_hours(),
-                remaining_time.num_minutes() % 60,
-                remaining_time.num_seconds() % 60,
-            )
-        } else {
-            (0, 0, 0)
-        };
+        let mut values = map!(
+            "icon" => Value::icon("tea"),
+        );
 
-        let mut widget = Widget::new().with_format(format.clone());
+        if is_timer_active {
+            widget.state = State::Info;
```

Comment:
The state change seems unrelated

Reply:
That's true. I was mirroring something I did in the pomodoro block (which also needed this formatter).
I can remove it from this PR if you'd like.

Reply:
Let's remove this for now, maybe add it later in a new PR.